### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by fabianhauser**
I think the commit messages speak for themselves :-)

### CODE REVIEW

File: nele/__init__.py [hunk 1]
```diff
@@ -119,16 +128,17 @@ def send_newsletter(source, config, recipients):
 def main():
     arguments = docopt(__doc__, version='Nele %s' % ___version___)
     source = arguments['<newsletter_source>']
-    config = yaml.load(open(arguments['--config']))
+    config = yaml.load(open(arguments['--config']), Loader=yaml.SafeLoader)
```
Comment (raphiz) [inline nele/__init__.py]: Ist das dasselbe wie 
```suggestion
    config = yaml.safe_load(open(arguments['--config']))
```
?

**Comment by raphiz** [comment]
:tada: Thanks for the PR! Will merge it - I just added a question for clarification

**Comment by fabianhauser** [comment]
@raphiz ready to be merged! 🙂

**Comment by raphiz** [comment]
Thank you :tada: Will make a new release later today

**Pull request merged**